ideascube/library/static/library/library.css

```diff
@@ -0,0 +1,42 @@
+/* ************************************************* */
```

Comment:
I see three new requests for "purecss" files, plus "main.css", plus the specific css, am I missing something?